src/test/java/catalinaTema5/DemoqaElements/RadioButtons.java

```diff
@@ -0,0 +1,22 @@
+package catalinaTema5.DemoqaElements;
+
+import catalinaTema5.BrowserManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class RadioButtons {
+    public static void main(String[] args){
+        goToRadioButtonsPage();
+ }
+
+    public static  void goToRadioButtonsPage() {
```

Comment:
This method throws an exception. Please double check how you identify the elements and make it pass. Thanks!